### PR TITLE
Feature/thermal rotation

### PR DESCRIPTION
**Pull request by noelbrownback**
Verified thermal camera is mounted with 8x8 text on drone right, cable coming up out of tilt assembly. 
Thermal camera in FPV view.

### CODE REVIEW

**Comment by NathanVaughn** [comment]
Any feelings on making this a setting, like 4 radio buttons that are "rotate 0/90/180/270" ?

**Comment by noelbrownback** [comment]
Dennis has specific build instructions for how we mount the camera. So I
would say let’s leave it.

File: GUI/app/tabs/thermal_view_control.py [hunk 1]
```diff
@@ -109,8 +110,13 @@ def update_canvas(self, pixels: List[int]) -> None:
             for p in pixels
         ]
 
+        # Rotate 90° to orient for mounting correctly
+        float_pixels_matrix = np.reshape(float_pixels, (8,8))
```
Comment (NathanVaughn) [inline GUI/app/tabs/thermal_view_control.py:114]: use `self.camera_x` and `self.camera_y` instead, so just there are less magic numbers

**Review by NathanVaughn — CHANGES_REQUESTED** [review]

File: GUI/app/tabs/thermal_view_control.py (hunk 1 above)
Comment (NathanVaughn) [reply GUI/app/tabs/thermal_view_control.py:114]: Otherwise, lgtm

Comment (noelbrownback) [reply GUI/app/tabs/thermal_view_control.py:114]: noted ill change that now.

**Review by NathanVaughn — APPROVED** [review]

**Review by dbaldwin — APPROVED** [review]
👍

**Pull request merged**